fava/application.py

```diff
@@ -429,7 +429,10 @@ def jump():
     qs_dict = url.decode_query()
     for key, values in request.args.lists():
         if len(values) == 1 and values[0] == "":
```

Comment:
I'd insist on catching exception here. This is the right way of deleting an item from a python dict, even if it's a single threaded program.